### DISCUSSION TITLE
Quality control of research paper section

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/awesome-green-software#30] [General]
We want to facilitate access to strong research output, while avoiding promotion of false claims. This is a relatively new list and as the list of research papers grows we need to be mindful of their quality. It is entirely possible that the community will curate it organically. Despite the [disclaimer](https://github.com/Green-Software-Foundation/awesome-green-software/issues/9) poor texts should not be included, and the pull request review process has the potential to work against this. However, the list grows relatively fast, and the pull request review process does not have enough scope for detailed review. In particular if texts that are not academically peer-reviewed are included in this section.

Additionally, some quality control process has the potential to curate a 'endorsed list' of papers, that are in certain ways fundamental to the GSF.

**Comment by atg-abhishek** [comment]
Thanks for creating this @dschien - I definitely agree with this as you raised it in the WG meeting. My suggestion would be that we explore this jointly with #14

**Comment by dtoakley** [comment]
Asim: maybe flagging papers a peer reviewed, published in a reputable journal, or some sort of other validation in the awesome list. 
Pierre: could also have a process (Website?) for people to star or vote on the items in the list
Asim: this must be a solved problem - let's look to other lists
